lib/ts/recipe/emailverification/recipe.ts

```diff
@@ -164,4 +186,28 @@ export default class Recipe extends RecipeModule {
     isErrorFromThisRecipe = (err: any): err is STError => {
         return STError.isErrorFromSuperTokens(err) && err.fromRecipe === Recipe.RECIPE_ID;
     };
+
+    getEmailForUserId = async (userId: string, userContext: any): Promise<string> => {
+        if (this.config.getEmailForUserId !== undefined) {
+            const email = await this.config.getEmailForUserId(userId, userContext);
```

Comment:
Does `getEmailForUserId` return a `Promise<string>`? So then the `if (email !== undefined) {` will always be true:
- How come TS did not complain about this?
- Should we make `getEmailForUserId` return a `Promise<string | undefined>` instead?